spec/factories/event.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :event do
+    start '2019-08-11 03:00:00'
+    finish '2019-08-11 20:00:00'
+    error nil
+    success true
+    created_at '2019-08-11 20:00:00'
```

Comment:
I'm not sure we would want all factory-created items locked in like this..

Reply:
yes. I will update this to Time.now and with a difference of 2 hours.